cfgov/jinja2/v1/_includes/macros/video-player.html

```diff
@@ -36,16 +36,18 @@
     %}
     {% set button_pos   = options.button_pos or 'center' %}
     {% set is_flexible  = ( options.video.width or options.video.height ) is not defined %}
-    <div class="video-player video-player__{{ video_player }}"
+    <div class="video-player
+                video-player__{{ video_player }}
+                {{'featured-content-module_visual' if options.video.is_fcm else '' }}"
          data-video-id="{{ video_id }}"
-         data-width="{{ options.video.width }}"
-         data-height="{{ options.video.height }}">
+         data-width="{{ options.video.width if options.video.width else '' }}"
```

Comment:
We should set these numbers to default to the youtube spacing or even just stay at the youtube spacing, the flexible box just doesn't work as well with the share and close button with the way the specs are designed. I'm happy to talk with the designers if we do want to make it work for flexible spacing

From: Jimmy Wilson [mailto:notifications@github.com]
Sent: Friday, February 26, 2016 02:58 PM
To: cfpb/cfgov-refresh <cfgov-refresh@noreply.github.com>
Cc: Munoz, Kimberly (CFPB)
Subject: Re: [cfgov-refresh] Featured Content Modules - Now with video! (#1499)


In cfgov/jinja2/v1/_includes/macros/video-player.html<https://github.com/cfpb/cfgov-refresh/pull/1499#discussion_r54314264>:

>           data-video-id="{{ video_id }}"
> -         data-width="{{ options.video.width }}"
> -         data-height="{{ options.video.height }}">
> +         data-width="{{ options.video.width if options.video.width else '' }}"


What happens if nothing is passed and these are both empty? Should we use default width and height?

—
Reply to this email directly or view it on GitHub<https://github.com/cfpb/cfgov-refresh/pull/1499/files#r54314264>.